### PR TITLE
Externalized the MATERIAL_LIST into a settings key

### PR DESCRIPTION
**Pull request by Denis-net**
Ciao,

Ho trasformato la MATERIAL_LIST nell' impostazione `oneMoreTimeThemeList`.

Facendo cosí chiunque puó impostare i propri temi preferiti,

Inoltre potrai rispondere all'issue #10 con:
prova ad impostare nei settings:
``` JSON
"oneMoreTimeThemeList": [ ]
```
e vedrai che cambierá in modo molto piú puntuale.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ A simple extension that chooses and applies a *random* theme.
 
 - `randomThemeSwitcher.themeList`: a list of string with labels of themes.
 
+- `oneMoreTimeThemeList`: a list of themes that will be kept one more time before being changed when randomized
```
Comment (zanza00) [inline README.md]: I think that like this sound better, we don't need to "expose" the internal of how the extension work. Also like this is more clear

```suggestion
- `preventReloadThemeList`: some themes, after they are applied, reload the instance. If are present in this list the extension will not set a new random theme after they are applied.
```

File: README.md [hunk 2]
```diff
@@ -62,6 +66,11 @@ example:
 
 ## Release Notes
 
+## 0.2.1
+
+- Externalized the MATERIAL_LIST into a more customizable `oneMoreTimeThemeList`
```
Comment (zanza00) [inline README.md]: change also here

File: src/extension.ts [hunk 3]
```diff
@@ -20,9 +20,10 @@ async function changeTheme(
   const userSettings = getUserSettings();
   const i = getRandomInt(themeList.length - 1);
   const newTheme = themeList[i];
+  const oneMoreTimeThemeList: string[] = <string[]>userSettings.get(SettingsKeys.OneMoreTimeThemeList, MATERIAL_LIST);
```
Comment (zanza00) [inline src/extension.ts]: I would change this name in something like `preventReloadThemeList` or something along those lines

File: README.md (hunk 1 above)
Comment (Denis-net) [reply README.md]: Hey @zanza00 !
Thanks for your replies:

I didn't understand what the problem was with the list of materials,
I thought you had preferences about that suite.😅
Now it's clear and I agree with you on the name.

**Comment by Denis-net** [comment]
I have renamed the list based on what you rightly pointed out in the Change request 👍🏼
May you have a nice day 😊👋🏼

**Pull request merged**